yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -924,9 +951,16 @@ func getUnsupportedFeaturesFromSchemaAnalysisReport(featureName string, issueRea
 			}
 			link = issue.DocsLink
 			objects = append(objects, objectInfo)
+			if !minVersionsFixedInSet {
```

Comment:
nit: we can use the `minVersionsFixedIn` to check if its set or not.

Reply:
Ah, so it would be wrong to check if `minVersionsFixedIn` is not nil. Consider the case where out of 2 issues, first one has `minVersionsFixedIn= nil` and second one has `minVersionsFixedIn as some not nil value`. This is a case where we want to catch and throw an error, because we want all issues in an unsupportedFeature to have the same `minVersionsFixedIn` (at least for now. Maybe when we change the report structure into a table, this might change). But if we just check if `minVersionsFixedIn` is nil or not, we will fail to catch this case.

Reply:
Ah okay got it.